### PR TITLE
Add currency logo next to pooled tokens on User Liquidity

### LINKED ISSUE #1339: Add token logo on User liquidity next to pooled tokens for clarity
**Description**
When checking user liquidity pooled tokens we have to read the title to know which token is what, in this example DAI and ETH. To make it easier to read we should add the token logo next to them.

image

#### Definition of done
- [x] Description is created (Task creator)
- [x] Code is complete (Developer)
- [ ] Scope of test and what was done section is defined in task (Developer)
- [ ] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

**Proposal**
Add the currency next to pooled tokens on user liquidity

### PR DESCRIPTION
**Pull request by Diogomartf**
Closes #1339

# Summary

Added token logo next to pooled tokens on User Liquidity to make it easy to read.
- updated UserLiquidity
- updated ValueWithLabel component to accept children
- rename Pool tokens to LP Tokens

image

  # To Test

1. Go to liquidity pools
2. Choose a pair
3. Check the logo next to pooled tokens on "Your liquidity" section

### CODE REVIEW

**Review by karczuRF — APPROVED** [review]

**Comment by niemam29** [comment]
Tested and looks good!

File: src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx [hunk 1]
```diff
@@ -70,14 +71,22 @@ export function UserLiquidity({ pair }: UserLiquidityProps) {
...
+            </Flex>
+          </ValueWithLabel>
+          <ValueWithLabel title={t('userLiquidity.pooledToken', { token: currency1?.symbol })}>
+            <Flex alignItems="center">
+              <Box mr="6px">
+                <CurrencyLogo size="14px" currency={currency1} />
+              </Box>
+              {token1Deposited ? token1Deposited.toSignificant(6) : '0'}
+            </Flex>
+          </ValueWithLabel>
```
Comment (berteotti) [inline src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx:89]: Proper style is missing. Value text style different from the other values.

**Comment by MilanVojnovic95** [comment]
Tested and looks ok to me 👍

File: src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx (hunk 1 above)
Comment (Diogomartf) [reply src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx:89]: Didn't understand what you mean with proper style.

Comment (Diogomartf) [reply src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx:89]: Got it, thanks!

Comment (Diogomartf) [reply src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx]: Fixed on 1e92cbc7b569298ee2ac5b719bd5a82a2f868fad

**Review by MilanVojnovic95 — APPROVED** [review]

**Review by berteotti — APPROVED** [review]
Good job!

**Pull request merged**